File: client/sms.js
const accountSid = 'ACbae95b1c54598d164d57372fd8c901dd'; // Your Account SID from twilio.com/console
const authToken = 'cb2820d5346912d24b185ddb7f9bd145'; // Your Auth Token from twilio.com/console
const client = require('twilio')(accountSid, authToken);

function sendOTPMobile(contactNumber, smsContent) {
  return new Promise((resolve, reject) => {
    client.messages
      .create({
        body: `Greetings, Please find your SMS:\n${smsContent}`,
        from: '+13149474018', // this is given by Twilio
        to: contactNumber,
      })
      .then((message) => {
        resolve({
          message: 'SMS sent successfully',
          statusCode: 200,
        });
      })
      .catch((error) => {
        reject({
          message: 'Something went wrong',
          error: error.message,
          statusCode: 500,
        });
      });
  });
}

const smsContent = 'Hello Alex Fandu';
sendOTPMobile('+919955262206', smsContent)
  .then((response) => console.log(response))
  .catch((error) => console.error(error));

File: client/sms.py
from twilio.rest import Client
from twilio.base.exceptions import TwilioRestException

# Twillio Documentation: [Twillio Python](https://www.twilio.com/docs/libraries/python)

# send_otp_message


def send_otp_mobile(contact_number, sms_content):
    # logger.info(f"send_otp_mobile: {contact_number}")
    response = {"message": "SMS sent successfully",
                "status_code": 200}
    try:
        # Find your Account SID and Auth Token at twilio.com/console
        # Your Account SID from twilio.com/console
        account_sid = "ACbae95b1c54598d164d57372fd8c901dd"
        # Your Auth Token from twilio.com/console
        auth_token = "cb2820d5346912d24b185ddb7f9bd145"
        client = Client(account_sid, auth_token)
        message = client.messages.create(
            to=contact_number,  # "+919955262206"
            from_="+13149474018",  # this is given by Twillio
            body=f"\nGreetings, Please find your SMS: \n{sms_content}"
        )

    except TwilioRestException as e:
        response["message"] = {"message": f"Something went wrong",
                               "error": str(e)}
        response["status_code"] = 500

    return response


sms_content = """Hello Alex Fandu"""
response = send_otp_mobile("+919955262206", sms_content=sms_content)
print(response)
